src/SFA.DAS.Provider.PR.Web/Controllers/AddEmployer/StartController.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.Provider.PR.Web.Authorization;
+using SFA.DAS.Provider.PR.Web.Models;
+
+namespace SFA.DAS.Provider.PR.Web.Controllers.AddEmployer;
+
+[Authorize(Policy = nameof(PolicyNames.HasProviderAccount))]
+
+[Route("/{ukprn}/addEmployer/start")]
+public class StartController : Controller
```

Comment:
Authorization along with policy is missing here

Reply:
fixed